applications/ExaquteSandboxApplication/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ message("**** configuring KratosExaquteSandboxApplication ****")
 include(pybind11Tools)
 
 include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/MeshingApplication )
```

Comment:
I would add [check](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/IgaApplication/CMakeLists.txt#L8-L15) if the MeshingApp is compiled to get a proper error message

Reply:
Done